operator/src/main/java/org/keycloak/operator/v2alpha1/crds/KeycloakStatusCondition.java

```diff
@@ -26,6 +25,7 @@
 public class KeycloakStatusCondition {
     public static final String READY = "Ready";
     public static final String HAS_ERRORS = "HasErrors";
+    public static final String ROLLING_UPDATE = "RollingUpdate";
```

Comment:
Not sure if `RollingRestart` is more correct, I'll check and eventually we can change while cleaning up.

Reply:
Well, doing restart to roll out an update. 🙂